tuning/utils/config_utils.py

```diff
@@ -87,3 +89,55 @@ def get_hf_peft_config(task_type, tuning_config):
         hf_peft_config = None  # full parameter tuning
 
     return hf_peft_config
+
+
+def post_process_job_config(job_config_dict):
```

Comment:
can we keep this function inside the build folder since its to do with KTO config only? these utils are actually for our tuning config structures https://github.com/foundation-model-stack/fms-hf-tuning/tree/main/tuning/config .

Things related to docker build and KTO, we can keep inside build folder as its not relevant to library users and contributors